mula/scheduler/server/server.py

```diff
@@ -229,9 +229,9 @@ def patch_scheduler(self, scheduler_id: str, item: models.Scheduler) -> Any:
     def list_tasks(
         self,
         request: fastapi.Request,
-        scheduler_id: Optional[str] = None,
-        type: Optional[str] = None,
-        status: Optional[str] = None,
+        scheduler_id: Union[str, None] = None,
```

Comment:
Why replace `Optional[str]` with `Union[str, None]`?

Reply:
Not sure, I got this after a merge